crispy_forms/templates/bootstrap4/field.html

```diff
@@ -37,6 +37,11 @@
                 {% include 'bootstrap4/layout/help_text_and_errors.html' %}
             {% elif field|is_file and use_custom_control  %}
                 {% include 'bootstrap4/layout/field_file.html' %}
+            {% elif field|is_selectsingle and use_custom_control %}
```

Comment:
I did wonder about moving this down inside the next div (I think it works being the same number of lines)

Reply:
I've changed it. My thoughts were between having one more nested `if` (after the change) or the similar pattern that we had for other fields.